<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget"/>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>400</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu_File">
    <property name="title">
     <string>&amp;File</string>
    </property>
    <addaction name="action_Save_Workspace"/>
    <addaction name="action_Load_Workspace"/>
    <addaction name="separator"/>
    <addaction name="actionE_xit"/>
   </widget>
   <widget class="QMenu" name="menu_CAN">
    <property name="title">
     <string>&amp;CAN</string>
    </property>
    <addaction name="action_Connect"/>
    <addaction name="action_canviewer"/>
   </widget>
   <widget class="QMenu" name="menu_Help">
    <property name="title">
     <string>&amp;Help</string>
    </property>
    <addaction name="action_About"/>
   </widget>
   <widget class="QMenu" name="menu_Device">
    <property name="title">
     <string>&amp;Device</string>
    </property>
    <addaction name="actionDevice_List"/>
   </widget>
   <addaction name="menu_File"/>
   <addaction name="menu_CAN"/>
   <addaction name="menu_Device"/>
   <addaction name="menu_Help"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionE_xit">
   <property name="text">
    <string>E&amp;xit</string>
   </property>
  </action>
  <action name="action_canviewer">
   <property name="text">
    <string>Bus &amp;Monitor</string>
   </property>
  </action>
  <action name="action_About">
   <property name="text">
    <string>&amp;About</string>
   </property>
  </action>
  <action name="action_Load_Workspace">
   <property name="text">
    <string>&amp;Load Workspace</string>
   </property>
  </action>
  <action name="action_Save_Workspace">
   <property name="text">
    <string>&amp;Save Workspace</string>
   </property>
  </action>
  <action name="actionDevice_List">
   <property name="text">
    <string>Device &amp;List</string>
   </property>
  </action>
  <action name="action_Connect">
   <property name="text">
    <string>&amp;Connect</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
